modules/basegl/include/modules/basegl/processors/instancerenderer.h

```diff
@@ -92,6 +95,8 @@ class IVW_MODULE_BASEGL_API InstanceRenderer : public Processor, public Property
     virtual void initializeResources() override;
     virtual void process() override;
 
+    std::optional<mat4> process(bool enableBoundingBoxCalc);
```

Comment:
For clarity I think this could be called render
```suggestion
/* Render all meshes. 
* @return The axis-aligned bounding box of all vertices if enableBoundingBoxCalc, otherwise nullptr.
    std::optional<mat4> render(bool enableBoundingBoxCalc);
```